src/core/OnchainKitConfig.ts

```diff
@@ -43,6 +43,9 @@ export const getOnchainKitConfig = <K extends keyof typeof ONCHAIN_KIT_CONFIG>(
 /**
  * Update the ONCHAIN_KIT_CONFIG object directly by providing the properties to update.
  * This is powerful when you use OnchainKit utilities outside of the React context.
+ * 
+ * @param {SetOnchainKitConfig} properties - Properties to update in the configuration
+ * @returns {Function} Returns the getOnchainKitConfig function for chaining
```

Comment:
Since we're using TypeScript, it's not necessary to include types in the JSDoc comments.

This can be changed to:
```
 * @param properties - Properties to update in the configuration
 * @returns The getOnchainKitConfig function for chaining
```